server/db/Sql.js

```diff
@@ -1,6 +1,6 @@
 const knex = require("knex");
 const Spinner = require("cli-spinner").Spinner;
-const _ = require("lodash");
+import _ from "lodash";
```

Comment:
The server is not a React app. It won't work this way. 
No changes are required there.